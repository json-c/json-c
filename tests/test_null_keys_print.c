#include <stddef.h>
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

#include "json.h"
#include "parse_flags.h"

int main(int argc, char **argv)
{
	struct json_object *new_obj;

	new_obj = json_tokener_parse("{ \"foo\\u0000bar\": \"qwerty\\u0000asdf\" }");
	printf("new_obj.to_string()=%s\n", json_object_to_json_string(new_obj));
	json_object_put(new_obj);

	return EXIT_SUCCESS;
}
