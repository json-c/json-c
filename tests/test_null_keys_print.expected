new_obj.to_string()={ "foo\u0000bar": "qwerty\u0000asdf" }
